dts/bindings/led_strip/ti,tlc59731.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Javad Rahimipetroudi <javad.rahimipetroudi@mind.be>
+# SPDX-License-Identifier: Apache-2.0
+
+compatible: "ti,tlc59731"
+description: TLC59731 RGB LED Controller
+
+properties:
+  gpios:
+    required: true
+    type: phandle-array
+    description: |
+      GPIO to send command data to the controller.
+  chain-length:
+    type: int
+    required: true
+    description: |
+      The number of devices in the daisy-chain.
```

Comment:
@simonguinot think it would make sense to have a base dts file wit the length property and have `include/zephyr/drivers/led_strip.h` validate the `num_pixels` parameter against it (outside the scope of this PR)

Reply:
Yes